src/app/components/dashboard-apps-dock/dashboard-apps-dock.html

```diff
@@ -0,0 +1,87 @@
+<div id="app-dock-container" class="hidden-xs">
+  <div id="app-dock">
+    <div app-deletion-modal=true></div>
+    <ul class="dock list-unstyled" ng-class="{minimized: appDock.isMinimized, animated: appDock.isAnimated()}">
+      <li ng-repeat="app in apps" ng-class="{active: isActiveApp(app.id)}" class="dock-icon">
+        <div class="title" tooltip="{{app.name.length > 14 ? app.name : null}}" , tooltip-placement="bottom">{{app.name | truncate:14:"...":true}}</div>
```

Comment:
Instead of truncate, we should use the ellipsis css property (cf. https://developer.mozilla.org/en/docs/Web/CSS/text-overflow)